Tasks/AzureFileCopy/AzureFileCopy.ps1

```diff
@@ -96,7 +83,7 @@ try
     Import-Module ./$azureUtility -Force
 
     # Getting connection type (Certificate/UserNamePassword/SPN) used for the task
-    $connectionType = Get-ConnectionType -connectedServiceName $connectedServiceName -distributedTaskContext $distributedTaskContext
+    $connectionType = Get-ConnectionType -connectedServiceName $connectedServiceName
```

Comment:
distributedTaskContext was primarily used for Get-ServiceEndpoint in this flow. The latest version of Get-VstsEndpoint doesn't require the task context.